packages/abi-coder/src/utils/utilities.test.ts

```diff
@@ -197,14 +197,55 @@ describe('Abi Coder Utilities', () => {
     };
     const BASE_OFFSET = 0;
     const DATA_OFFSET = 0;
-    // prettier-ignore
     const EXPECTED = new Uint8Array([
-      0,       0,       0,       0,       0,       0,       0,       0,
-      0,       0,       0,       0,       0,       0,       0,       2,
-      0,       0,       0,       0,       0,       0,       0,       2,
-      0,       0,       0,       0,       0,       0,       0,       48,
-      0,       0,       0,       0,       0,       0,       0,       1,
-      0,       0,       0,       0,       0,       0,       0,       1,
+      0,
```

Comment:
The previous (manual?) formatting for this was better and it'd be good to leave it as it was.